rtloader/include/rtloader_types.h

```diff
@@ -121,6 +121,10 @@ typedef void (*cb_log_t)(char *, int);
 typedef void (*cb_set_check_metadata_t)(char *, char *, char *);
 // (hostname, source_type_name, list of tags)
 typedef void (*cb_set_external_tags_t)(char *, char *, char **);
+// (key, value)
+typedef void (*cb_store_value_t)(char *, char *);
+// (value)
+typedef void (*cb_retrieve_value_t)(char *, char **);
```

Comment:
We should simply return the value instead of taking a `char **`.

Reply:
I don't think we can do that in cgo.